tests/prompts/test_stream.py

```diff
@@ -31,27 +32,59 @@ def gpt4_behaviour():
             ),
         ]
     )
+    
```

Comment:
can you please add in github env these secrets?

Reply:
I don't have authority to do that :(

Reply:
done!